### PR TITLE
Add info card to home

### PR DESCRIPTION
**Pull request by jgfidelis**
https://card2--storecomponents.myvtex.com/

I appreciate any help on making it cooler from the great front end guys here. If you think this is not necessary, I can close it.

#### Types of changes

* [ ] Bug fix (a non-breaking change which fixes an issue)
* [x] New feature (a non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: store/blocks.json [hunk 1]
```diff
@@ -92,6 +93,20 @@
...
+  "info-card": {
+    "props": {
+      "isFullModeStyle": true,
+      "textPosition": "left",
+      "headline": "Enjoy the outside with style",
+      "subhead": "Choose from a selection of our best Panama Hats",
+      "callToActionMode": "button",
+      "callToActionText": "Shop Panama Hats",
+      "callToActionUrl": "hats/panama",
+      "imageUrl": "https://i.ibb.co/qYktksq/resized.png",
```
Comment (lbebber) [inline store/blocks.json]: 🤔 how about hosting this on vtex servers?

File: styles/css/vtex.store-components.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
+.infoCardContainer {
+  height: 500px;
+  background-position: top center;
```
Comment (matheusps) [inline styles/css/vtex.store-components.css]: Is this rule really necessary? 🤔

File: styles/css/vtex.store-components.css [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+@media only screen and (min-width: 980px) {
+  .infoCardContainer {
```
Comment (matheusps) [inline styles/css/vtex.store-components.css]: You did something very similar to the global declaration ... why did it again?

File: styles/css/vtex.store-components.css [hunk 4]
```diff
@@ -0,0 +1,22 @@
+.infoCardContainer {
+  height: 500px;
```
Comment (matheusps) [inline styles/css/vtex.store-components.css]:
```suggestion
  height: 31.25rem;
```

File: styles/css/vtex.store-components.css [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  background-position: top center;
+}
+
+@media only screen and (min-width: 980px) {
```
Comment (matheusps) [inline styles/css/vtex.store-components.css]:
```suggestion
@media only screen and (min-width: 60em) {
```

File: styles/css/vtex.store-components.css [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+    background-position: top center;
+  }
+
+  .infoCardTextContainer {
+    margin-left: 6rem;
+  }
+
+}
+
+@media only screen and (max-width: 490px) {
```
Comment (matheusps) [inline styles/css/vtex.store-components.css]:
```suggestion
@media only screen and (max-width: 30em) {
```

Comment (matheusps) [reply styles/css/vtex.store-components.css]: 30em matches tachyons `-ns`.

File: styles/css/vtex.store-components.css (hunk 5 above)
Comment (matheusps) [reply styles/css/vtex.store-components.css]: 60em matches tachyons `-l`

File: store/blocks.json (hunk 1 above)
Comment (jgfidelis) [reply store/blocks.json]: will take a look on how to do this

**Pull request closed without merging**